book/_config.yml

```diff
@@ -45,6 +45,8 @@ execute:
     - "**/earthaccess.ipynb"
     - "**/icepyx.ipynb"
     - "**/sliderule.ipynb"
+    - "**/icesat-2-land-ice-products.ipynb"
+    - "**/sliderule-feature-find.ipynb"
```

Comment:
If you want to commit these notebooks with outputs that would be fine since they are not being executed. that way outputs will show up on the website

Reply:
Good idea! I'll get that done so the figures made there. Also sorry for all the build failures!

Reply:
Is it okay that i put the PDF of the slides here as a download option?

Reply:
P.S. sorry for being late with all this

Reply:
all good